Automattic/wp-calypso
# Add instruction modal for locked google domains

> [!WARNING]  
> Dependent on this PR: https://github.com/Automattic/wp-calypso/pull/80007

Fixes https://github.com/Automattic/dotcom-forge/issues/3295

## Proposed Changes

* Add instruction modal on how to unlock

https://github.com/Automattic/wp-calypso/assets/6586048/3a1391e1-37a3-469f-a6d7-e3eb8144697b


## Testing Instructions



* Go to `/setup/google-transfer/domains`
* Enter a locked domain
* See if "How to unlock" link renders

## Pre-merge Checklist
